app/views/IndexView.scala.html

```diff
@@ -1,31 +0,0 @@
-@*
- * Copyright 2024 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *@
-
-@this(
-    layout: templates.Layout
-)
-
-@()(implicit request: Request[_], messages: Messages)
-
-@layout(
-    pageTitle    = titleNoForm(messages("index.title")),
-    showBackLink = false
-) {
-
-    <h1 class="govuk-heading-l">@messages("index.heading")</h1>
-
-    <p class="govuk-body">@messages("index.guidance")</p>
-}
```

Comment:
We can also delete following from messages.en 
`index.title = trader-goods-profiles-frontend
index.heading = trader-goods-profiles-frontend
index.guidance = Welcome to your new frontend. Please see the README file for a guide to getting started.`

Reply:
agreed

Reply:
fixed